GDM.HW4.OOP.Classes.Task3/Program.cs

```diff
@@ -0,0 +1,371 @@
+            {
+                Console.WriteLine("Invalid command");
+                ConsoleOptionMenu();
+            }
+            while (response != "")
+            {
+                switch (response)
+                {
+                    case "0":
+                        Environment.Exit(0);
+                        break;
+
+                    case "1":
+                        SetNewStoreAddress();
+                        SetStoreCapasity();
+                        ConsoleOptionMenu();
+                        break;
+
+                    case "2":
+                        SetNewPhoneModelNameAndPrice();
+                        ConsoleOptionMenu();
+                        break;
+
+                    case "3":
+                        GetPhonesInStore();
+                        ConsoleOptionMenu();
+                        break;
+
+                    case "4":
+                        Console.Clear();
+                        ConsoleOptionMenu();
+                        break;
+
+                    default:
+                        Console.WriteLine("-> Invalid command");
+                        ConsoleOptionMenu();
+                        break;
+                }
+            }
+         }
+        public static void SetShopName()
+        {
+            Console.WriteLine("Please write valid (not empty) name for shop");
+            string shopName = Console.ReadLine();
+            _shop = new Shop();
+
+            if (shopName.Length < 1)
+            {
+                SetShopName();
+            }
+            else
+            {
+                _shop.ShopName = shopName;
+                Console.WriteLine($"-> Great, the name of shop is '{_shop.ShopName}'");
+            }
+        }
+        public static void SetNumberOfStores()
+        {
+            Console.WriteLine($"Please write valid (number > 0 && number <= 10) number of stores for shop '{_shop.ShopName}'.");
+            string storesNumber = Console.ReadLine();
+            int tempNumberNumeric;
+            if (IsDigitsOnly(storesNumber) != true)
+            {
+                SetNumberOfStores();
+            }
+            else
+            {
+                tempNumberNumeric = Int32.Parse(storesNumber);
+                if (tempNumberNumeric < 1 || tempNumberNumeric > 10)
+                {
+                    SetNumberOfStores();
+                }
+                else
+                {
+                    _shop.NumberOfStores = tempNumberNumeric;
+                    Console.WriteLine($"-> Great, the number of stores in shop '{_shop.ShopName}' is '{_shop.NumberOfStores}'.");
+                }
+            }
+        }
+        public static void GetPhonesInStore()
+        {
+            Console.WriteLine($"-> Store in ''{_shop.ShopName}'' ");
+            for (int i = 0; i < _mobStore.Length; i++)
+            {
+                string storeCellStatus = "";
+                if (IsObjectInArrayNull(_mobStore, i) == true)
```

Comment:
ACHTUNG!!!! we don't compare booleans. 
Write: **if(bool){...}**  instead of **if(bool == bool){...}**